walletunlocker/service.go

```diff
@@ -287,6 +287,7 @@ func (u *UnlockerService) UnlockWallet(ctx context.Context,
 	walletUnlockMsg := &WalletUnlockMsg{
 		Passphrase:     password,
 		RecoveryWindow: recoveryWindow,
+		Wallet:         unlockedWallet,
```

Comment:
wallet must be unloaded in case the macaroon check above fails.

Reply:
Uh, thanks! I missed this one...